SystemTestPlan.md

```diff
@@ -0,0 +1,152 @@
+# System Test
+
+# Second Factor
+
+# Milestone 2
+
+#
+
+## System testing op interactie met Arduino
+_Hiermee testen we de requirements van: Er wordt een een bevestiging van actieve verbinding getoond, er is een manier om een kleurencode in te geven, te versturen en te wissen.
+
+### Test 1
+
+**Instructie:**
+Sluit de Arduino aan wanneer de server actief is. 
+
+**Verwacht resultaat:**
+Zodra de Arduino aan staat, gaat het ledje branden ter bevestiging van een actieve verbinding. 
+
+**Daadwerkelijk resultaat:**
+...
+
+
+### Test 2
+
+**Instructie:**
+Schakel de Arduino in wanneer de server inactief is.
+
+**Verwacht resultaat:**
+Zodra de Arduino aan staat, blijft het ledje uit ter bevestiging dat er geen actieve verbinding is. 
+
+**Daadwerkelijk resultaat:**
+...
+
+
+### Test 3
+
+**Instructie:**
+Druk drie keer de 'blauwe' knop in. 
+
+**Verwacht resultaat:**
+Er gaan drie blauwe (RGB)-leds branden.
+
+**Daadwerkelijk resultaat:**
+...
+
+
+
+### Test 4
+
+**Instructie:**
+Druk drie keer de 'rode' knop in.
+
+**Verwacht resultaat:**
+Er gaan drie rode (RGB)-leds branden.
+
+**Daadwerkelijk resultaat:**
+...
+
+
+### Test 5
+
+**Instructie:**
+Druk drie keer de 'groene' knop in.
+
+
+**Verwacht resultaat:**
+Er gaan drie groene (RGB)-leds branden.
+
+**Daadwerkelijk resultaat:**
+...
+
+
+### Test 6
+
+**Instructie:**
+Druk de 'groene', 'rode', en 'blauwe' knoppen ieder 1x in op de Arduino.
+
+**Verwacht resultaat:**
+Er gaan op de Arduino een groene, een rode en een blauwe (RGB)-led branden.
+
+**Daadwerkelijk resultaat:**
```

Comment:
Ik denk dat dit toch moet worden aangepast? ik vind dat test 6 genoeg is?

Reply:
Daar valt zeker wat voor te zeggen. Maar willen we niet in onze testen volledig vastleggen dat elke (RGB)led elke kleur goed kan weergeven?

Reply:
Ik denk dat het verstandig is om test 3,4,5 erin te houden. Je wilt kunnen testen of de ledjes op de juiste manier functioneren.

Reply:
ben ik het mee eens beter uitgebreid en misschien onnodig dan onvolledig.